src/Microsoft.DotNet.Tools.Compiler.Csc/Program.cs

```diff
@@ -149,9 +158,10 @@ private static IEnumerable<string> TranslateCommonOptions(CommonCompilerOptions
                 commonArgs.Add("-delaysign");
             }
 
-            // TODO: What is this? What does it mean to sign without a key?
-            // Is this "OSS" signing?
-            // if (options.StrongName)
+            if (options.PublicSign == true)
+            {
+                commonArgs.Add("-publicsign");
+            }
```

Comment:
I thought I got this right...

The keyfile section only adds if public sign is null, but publicsign adds if true, so you only get one copy in either case,right?